pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -18,6 +18,9 @@
 
 import org.checkerframework.checker.nullness.qual.Nullable;
 import org.checkerframework.checker.nullness.qual.PolyNull;
+import sun.util.calendar.BaseCalendar;
```

Comment:
We should never ever use "sun." internal classes in a public library like Postgres. This will also fail if used with the Java module system!

Reply:
Ya, I was going to get to that today.